files/demo.hledger

```diff
@@ -0,0 +1,178 @@
+﻿# Welcome to hledger, This file is where your financial history will be recorded.
+# It is a plain text file, so it is easy to edit and manipulate, and you can use
+# search and replace functionality in your text editor
+
+# Everything after '#', ';' or '*' are comments, so they are not read by hledger,
+# except in some cases to be discussed later
+
+
+# To start configuring our journal, it is a good idea to define a default currency
+D USD 1,000.00  ; Format - D Currency amountFormat
+
+
+# Now let's declare our main accounts. The order they appear here is the other they will
+# show on the reports. We need to tell hledger what is the name of each account using the
+# following code: A=Asset, L=Liabilities, R=Revenues, X=Expenses, E=Equity.
+# We will use the default name, but you can change them if you like
+
+# Attention! Here and in some other places you may need at least 2 spaces, instead of 1
+
+account Asset                  A  ; Format - account name  Code  <- 2 spaces at least between name and code
+account Liabilities            L
+account Revenue                R
+account Expenses               X
+account Equity                 E
+
+
+# Let's declare some other accounts, so they will appear in the desired order
+# You can organize them in a tree structure using ':' to indicate a a parent-child relation account
+
+account Asset:Cash
+account Asset:Investments
+account Asset:Real Estate
+account Liabilities:Credit Card
+account Liabilities:Mortgage
+
+
+# Possibly you will not start from scratch. You will want to import your data from a previous software that you
+# want to adapt for the powerful hierarchical account system hledger provides. For that we will use alias so that
+# whenever hledger sees an old account, it will substitute for its new name
+
+alias jpm=Asset:Bank:JP Morgan Chase                     ; Format - alias OLD=NEW
+alias visa=Liabilities:Credit Card:Visa Platinum         ; Substitute 'visa' for 'Liabilities:Credit Card:Visa Platinum'
+
+
+# Now we will finally start adding transactions to our ledger. For that, you will need to understand the format. Don't
+# worry if it seems complicated. You will quickly be used to it, and hledger offers some great tools to make that
+# process easier with an interactive filling app with auto-complete and tools to automatically import data from your
+# bank or any other source. It may even automatically fill the data for you if it finds a similar older transaction!
+# If You you are curious, google hledger-iadd or hledger-autosync for some examples
```

Comment:
s/You you/you/

"hledger-autosync" doesn't really exist, I removed it from the docs yesterday. ledger-autosync probably still works but I'm not sure it's notable enough to be singled out these days.

Maybe mention the built in add command as well, since they already have it, and hledger-web

Reply:
In fact I meant ledger-autosync, but that's fine. I will remove it. I will mention 'hledger add' and the hledger-web I think it is better somewhere else that I will show you. I will keep this unresolved until you decide if I mention it here or not

Reply:
Your call